File: LATIHAN1DPBO2025C2/Python/program/PetShop.py
# class declaration
class PetShop:

    # atribut private
    __id = ""
    __nama_produk = ""
    __kategori_produk = ""
    __harga_produk = ""

   # konstruktor
    def __init__(self, id="", nama_produk="", _kategori_produk="", harga_produk=""):
        self.__id = id
        self.__nama_produk = nama_produk
        self.__kategori_produk = _kategori_produk
        self.__harga_produk = harga_produk

    # Getter and Setter

    def get_id(self):
        return self.__id

    def set_id(self, id):
        self.__id = id

    def get_nama_produk(self):
        return self.__nama_produk

    def set_nama_produk(self, nama_produk):
        self.__nama_produk = nama_produk

    def get_kategori_produk(self):
        return self.__kategori_produk

    def set_kategori_produk(self, kategori_produk):
        self.__kategori_produk = kategori_produk

    def get_harga_produk(self):
        return self.__harga_produk

    def set_harga_produk(self, harga_produk):
        self.__harga_produk = harga_produk

File: PHP/program/PetShop.php
<?php
// deklarasi kelas petshop
class PetShop {
    // atribut private
    private $id = '';
    private $nama_produk = '';
    private $kategori_produk = '';
    private $harga_produk = '';
    private $foto = '';

    // konstruktor
    public function __construct( $id = '', $nama_produk = '' , $kategori_produk = '', $harga_produk = '',$foto='' ) {
        $this->id = $id;
        $this->nama_produk = $nama_produk;
        $this->kategori_produk = $kategori_produk;
        $this->harga_produk = $harga_produk;
        $this->foto = $foto;
    }
 /* Getter dan Setter */
    public function set_nim($id){
        $this->id = $id;
    }

    public function get_id(){
        return $this->id;
    }
    
    public function set_nama_produk($nama_produk){
        $this->nama_produk = $nama_produk;
    }
    public function get_nama_produk(){
        return $this->nama_produk;
    }
    public function set_kategori_produk($kategori_produk){
        $this->kategori_produk = $kategori_produk;
    }

    public function get_kategori_produk(){
        return $this->kategori_produk;
    }
    
    public function set_harga_produk($harga_produk){
        $this->harga_produk = $harga_produk;
    }
    public function get_harga_produk(){
        return $this->harga_produk;
    }
    public function set_foto($foto){
        $this->foto = $foto;
    }
    public function get_foto(){
        return $this->foto;
    }

}

?>